app/AppLoader.scala

```diff
@@ -48,6 +47,12 @@ class AppLoader extends ApplicationLoader with Logging {
 
     val contextWithConfig = context.copy(initialConfiguration = combinedConfig)
 
-    new AppComponents(contextWithConfig).application
+    try {
+      new AppComponents(contextWithConfig).application
+    } catch {
+      case t: Throwable =>
+        log.error("Unable to initialise prism", t)
+        throw t
```

Comment:
Whilst debugging this work we had to go onto the instance and run `$ journalctl -u prism` to get the error that helped us understand why prism was not starting. In order for this to not happen again, we're catching such throwables and logging them out to central ELK for easier debugging.